src/panels/lovelace/hui-root.ts

```diff
@@ -573,6 +573,12 @@ class HUIRoot extends LitElement {
         }
         newSelectView = index;
       }
+
+      // Check for requested edit mode
+      const urlParams = new URLSearchParams(window.location.search);
```

Comment:
We have a helper for this

Reply:
I going hunting for that one then 👍 

Also, should this URL option stay a completely manual one or could it make sense to add that to URL when the user enters the edit mode (manually via the UI) so that then a page refresh will keep them in edit mode?

Reply:
We should at least remove it when you exit edit mode.

I would say add it when entering edit mode.

Reply:
https://www.github.com/home-assistant/frontend/tree/dev/src%2Fcommon%2Furl%2Fsearch-params.ts

Reply:
I implemented the automatic "edit" search parameter adding and removing.

It is however not carried over when switching the tabs in edit mode. Should this be done?